### ISSUE TITLE
Light issue

### ISSUE THREAD

**Issue by Aslaak** [GlowstoneMC/Glowstone#401]
Hello !
I noticed this problem, I don't found any previous issues about it, so here it is:
https://youtu.be/fAImXL54Juk
( I'm running last build, but I already got it on build #294 )

**Comment by BSFishy** [comment]
Are you 100% sure that is happening with other people. To me, that looks like a client-side glitch.

**Comment by gdude2002** [comment]
This seems to be a combination of client-side and server-side issues.

That said, I have no idea what's causing them.

**Comment by BSFishy** [comment]
My server is running 1.11-SNAPSHOT-"ab9e3fd"-MC1.11 (Or whatever the latest is I think), and I am having no issues. It could have something to do with texture packs, or client-side mods.

**Comment by gdude2002** [comment]
Nah, I've definitely noticed unusual lighting issues with Glowstone in the past. Let's wait for @mastercoms.

**Comment by mastercoms** [comment]
I don't think server side lighting calculations have been implemented at all.

**Comment by Aslaak** [comment]
For answer, I just have optifine as mods and I don't see any changement without it, and it's the same for the texture-pack, the issue is the same with default texture-pack :/
Thank you for helping :)
